lib/gettext.ex

```diff
@@ -642,8 +615,23 @@ defmodule Gettext do
         end
 
       _other ->
+        # TODO: remove this once we stop supporting the old way of defining backends.
+        IO.warn("""
+        defining a Gettext backend by calling
+
+            use Gettext, otp_app: ..., ...
+
+        is deprecated. To define a backend, call:
+
+            use Gettext.Backend, otp_app: :my_app
+
+        Then, to use the backend, call this in your module:
+
+            use Gettext, backend: MyApp.Gettext
+
+        """)
```

Comment:
```suggestion
        """, Macro.Env.from_stacktrace(__CALLER__))
```

From Elixir v1.14, you can also just pass `__CALLER__`.

Reply:
Nice!